app/models/keyword_facet.rb

```diff
@@ -51,7 +51,7 @@ def value_fragments
         keyword_fragments << {
           'label' => keyword,
           'parameter_key' => key,
-          'name' => 'keywords',
+          'name' => 'q',
```

Comment:
Could `key` be used here instead so that there's no need to change both places?  Or does `key` and this `name` refer to different things?